Source/AnimatableImageView.swift

```diff
@@ -28,10 +75,16 @@ public class AnimatableImageView: UIImageView {
   ///
   /// - parameter data: GIF image data.
   public func prepareForAnimation(imageData data: NSData) {
-    image = UIImage(data: data)
     animator = Animator(data: data, size: frame.size, contentMode: contentMode, framePreloadCount: framePreloadCount)
-    animator?.prepareFrames()
-    attachDisplayLink()
+    
+    if let animator = animator {
+      animator.needsFramesResizing = self.needsFramesResizing
+      animator.prepareFrames()
+      posterImage = animator.animatedFrames.count > 0 ? animator.animatedFrames[0].image : nil
+      image = posterImage
+      delegate?.animatableImageView?(self, didUpdateFrameToIndex: 0)
```

Comment:
Hah, I know, it looks strange, but I did this according to the official Apple's guide - [link](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Protocols.html#//apple_ref/doc/uid/TP40014097-CH25-ID284)

Here is the quote: 

> You check for an implementation of an optional method by writing a question mark after the name of the method when it is called, such as someOptionalMethod?(someArgument).